packages/cli/src/controller/publish-controller.spec.ts

```diff
@@ -38,7 +38,7 @@ const projectSpecV0_2_0: ProjectSpecV0_2_0 = {
 
 const ipfsEndpoint = 'http://localhost:5001/api/v0';
 //Replace your access token before test
-const testAuth = 'MTA0MzE2NTc=JIwMq1cCzGIWddlskYRE';
+const testAuth = 'MTA0MzE2NTc=9ykFgFWle8ks8x7G7AgB';
```

Comment:
Can you read this from env? so we can hide it from pubic even it is just a dev env key.